sorting/gnome_sort.cpp

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2020 @author beqakd
+ * @file
+ * A basic implementation of gnome sort algorithm.
+ */
+
+#include <iostream>  // for io operations
+
+/**
+ * Copyright 2020 @author beqakd
```

Comment:
some part of this comment is repeated, no need to put it twice

Reply:
Fixed ^_^